VNScripts/CharacterView.gd

```diff
@@ -0,0 +1,64 @@
+extends Node2D
+
+var currentCharacters = []
+
+var characterMap = {
+	"thief": preload("res://RealElvenThief.tscn"),
+	"cat": preload("res://CatDude.tscn")
+}
+
+var exampleCharacterList1 = ["thief", "cat"]
+var exampleCharacterList2 = ["cat", "thief"]
+
+onready var positions = [
+	
+	[
+		#positions[0][0]
+		$"1CharacterPosition1".position
```

Comment:
Maybe call this 1CharacterPosition0 for less confusion? Then you can also drop to comments here, I think